ddapm_test_agent/trace.py

```diff
@@ -17,6 +18,9 @@
 from typing_extensions import TypedDict
 
 
+log = logging.getLogger(__name__)
```

Comment:
```suggestion
```

this module provides fuctional utility methods, there shouldn't be any logging done. If error information is required then the exceptions can be caught by the caller of these functions and logged there.